src/Umbraco.Web.UI.Client/src/installer/steps/user.html

```diff
@@ -46,17 +46,15 @@
 
 				<div class="control-group">
 					<div class="controls">
-						<label class="checkbox" for="subscribeToNewsLetter">
+						<label class="checkbox">
```

Comment:
Doesn't really need for attribute when wrapped inside a label - unless there were multiple controls and it shouldn't focus first one.